deploy/role.yaml

```diff
@@ -40,6 +40,19 @@ rules:
   - statefulsets/finalizers
   verbs:
   - '*'
+- apiGroups:
+  - "route.openshift.io"
+  resources:
+  - routes
+  - routes/custom-host
+  verbs:
+  - '*'
+- apiGroups:
+  - "networking.k8s.io"
```

Comment:
@rigazilla just last points, do we need double quotes for apiGroups and can we restrict verbs instead of grant all access?

Reply:
> @rigazilla just last points, do we need double quotes for apiGroups?

yeah they are useless

> and can we restrict verbs instead of grant all access?

that would be correct, but we're basically allowing all verbs for all the other resources. How about planning a fine tuning on all the roles/verbs in a separate issue?

Reply:
Yes, @rigazilla it would be nice to do with separate issue as looks like we have much more rights that needed